packages/design-system/src/MenuDivider/MenuDivider.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import MenuDivider from "./index";
+
+export default {
+  title: "Design System/Menu Divider",
+  component: MenuDivider,
+} as ComponentMeta<typeof MenuDivider>;
+
+export const MenuDividerExample = (
```

Comment:
nit: Alternatively you can try storybook decorators

Reply:
I just learnt about them while reviewing your code! Will update to include this in the next PR :)